src/app.js

```diff
@@ -0,0 +1,285 @@
+    // eslint-disable-next-line no-console
+    console.log('Vue Global Warn Handler', { error, vm, trace })
+}
+
+// Catch the rest
+window.onerror = function(error) {
+  if (process.env.NODE_ENV === 'development')
+    // eslint-disable-next-line no-console
+    console.log('Window Error Handler', error)
+  LogRocket.captureException(error)
+}
+
+// Add Filters
+Vue.filter('duration', duration)
+Vue.filter('shorten', shorten)
+Vue.filter('filterOnePercent', filterOnePercent)
+
+Vue.filter('roundWhole', roundWhole)
+Vue.filter('roundTenths', roundTenths)
+Vue.filter('roundHundredths', roundHundredths)
+Vue.filter('roundThousandths', roundThousandths)
+Vue.filter('roundTens', roundTens)
+Vue.filter('roundHundreds', roundHundreds)
+Vue.filter('roundThousands', roundThousands)
+
+Vue.component('HeightTransition', TransitionHeight)
+Vue.component('Truncate', TruncatedSpan)
+Vue.component('GetCloud', GetCloud)
+
+// This is a global mixin used to clean up any
+// references a component may have after it's destroyed.
+// It's a pretty heavy-handed approach to what we're trying
+// to accomplish but Vue doesn't seem to be removing all references
+// itself, which is leading to large memory leaks on data-heavy pages
+// as Apollo polls and when navigating between pages.
+// This could be more elegant and may be fixed by upgrading to Vue 3.
+const blockedProps = [
+  '$data',
+  '$listeners',
+  'theme',
+  '$attrs',
+  '$props',
+  'timeline',
+  'windowGroup',
+  'expansionPanel',
+  'expansionPanels',
+  'tabsBar',
+  'listItemGroup',
+  'btnToggle'
+]
+
+Vue.mixin({
+  destroyed() {
+    const prepForGC = () => {
+      try {
+        this.$el?.remove()
+        this.elm = null
+        this.$el = null
+        this.parent = null
+        this.$parent = null
+        this.data = null
+        // this.$data = null // Unsetting this throws internal Vue errors
+        // this.options = null // Unsetting this throws internal Vue errors
+        this.$options = null
+        this.$vnode = null
+        this.listeners = null
+        // this.$listeners = null // Unsetting this throws internal Vue errors
+        this._vnode = null
+        this._watcher = null
+        this._watchers = null
+        this._computedWatchers = null
+        this.$slots = null
+        this.slots = null
+        this.$scopedSlots = null
+        this.scopedSlots = null
+        this.$children = null
+        this.children = null
+        this.store = null
+        this.$store = null
+
+        // $apollo has no setter so we clear props instead
+        if (this.$apollo) {
+          this.$apollo.vm = null
+        }
+
+        Object.keys(this).forEach(key => {
+          try {
+            !blockedProps.includes(key) &&
+              !(key in this[key]?.$props) &&
+              (this[key] = null)
+          } catch {
+            /* */
+          }
+        })
+      } catch (e) {
+        // eslint-disable-next-line no-console
```

Comment:
Do we need to log this in all environments?

Reply:
Hm probably not, this is carried over from the existing garbage collector.